src/test/ui/const-generics/min_const_generics/const-expression-suggest-missing-braces-without-turbofish.rs

```diff
@@ -0,0 +1,49 @@
+#![feature(min_const_generics)]
+
+fn foo<const C: usize>() {}
+
+const BAR: usize = 42;
+
+fn a() {
+    foo<BAR + 3>(); //~ ERROR comparison operators cannot be chained
+}
+fn b() {
+    foo<BAR + BAR>(); //~ ERROR comparison operators cannot be chained
+}
+fn c() {
+    foo<3 + 3>(); //~ ERROR comparison operators cannot be chained
```

Comment:
Is the missing closing brace here on purpose?

Reply:
Nope, that was accidental. Thanks.